content/sitemap.xml

```diff
@@ -265,6 +265,138 @@
 <lastmod>2019-12-30T14:59:57-05:00</lastmod>
 </url>
 <url>
+<loc>http://0.0.0.0:4000/cn/docs/0.5.2-s3_hoodie.html</loc>
+<lastmod>2019-12-30T14:59:57-05:00</lastmod>
+</url>
+<url>
+<loc>http://0.0.0.0:4000/docs/0.5.2-s3_hoodie.html</loc>
+<lastmod>2019-12-30T14:59:57-05:00</lastmod>
+</url>
+<url>
+<loc>http://0.0.0.0:4000/cn/docs/0.5.2-gcs_hoodie.html</loc>
+<lastmod>2019-12-30T14:59:57-05:00</lastmod>
+</url>
+<url>
+<loc>http://0.0.0.0:4000/docs/0.5.2-gcs_hoodie.html</loc>
+<lastmod>2019-12-30T14:59:57-05:00</lastmod>
+</url>
+<url>
+<loc>http://0.0.0.0:4000/cn/docs/0.5.2-migration_guide.html</loc>
```

Comment:
is this needed?

Reply:
It seems that it's an auto-generated HTML page for every released version.